### PR TITLE
Generate schemas from JSON

### PR DESCRIPTION
**Pull request by corporateanon**
### Added
 - a script for schemas generation, which imports the schemas from the main repository https://github.com/nalgeon/iuliia
   - to execute it, run: `npm run generate`

### Fixed
 - gost_779_alt (Cyrillic `е` replaced with Latin `e`)
 - yandex_maps (`prev_mapping` added)

### CODE REVIEW

File: src/generated/gost_779_alt.ts [hunk 1]
```diff
@@ -33,22 +33,17 @@ export default {
         ъ: "``",
         ы: "y`",
         ь: "`",
-        э: "е`",
+        э: "e`",
```
Comment (corporateanon) [inline src/generated/gost_779_alt.ts:36]: Strange

File: src/generated/yandex_maps.ts [hunk 2]
```diff
@@ -37,13 +37,13 @@ export default {
         ю: "yu",
         я: "ya",
     },
-    prev_mapping: null,
+    prev_mapping: { е: "ye", ае: "ye", ие: "ye", ое: "ye", уе: "ye", эе: "ye", юе: "ye", яе: "ye" },
```
Comment (corporateanon) [inline src/generated/yandex_maps.ts:40]: Strange

File: src/generated/yandex_maps.ts [hunk 3]
```diff
@@ -37,13 +37,13 @@ export default {
...
     next_mapping: { ъе: "y" },
     ending_mapping: { ый: "iy" },
     samples: [
         [
             "Юлия, съешь ещё этих мягких французских булок из Йошкар-Олы, да выпей алтайского чаю",
-            "Yuliya, syesh eschyo etikh myagkikh frantsuzskikh bulok iz Yoshkar-Oly, da vypey altayskogo chayu",
+            "Yuliya, syesh yeschyo etikh myagkikh frantsuzskikh bulok iz Yoshkar-Oly, da vypey altayskogo chayu",
```
Comment (corporateanon) [inline src/generated/yandex_maps.ts:46]: Strange

File: src/generated/gost_779_alt.ts [hunk 4]
```diff
@@ -33,22 +33,17 @@ export default {
...
-        цй: "c",
-        цы: "c",
-    },
+    next_mapping: { це: "c", ци: "c", цй: "c", цы: "c" },
     ending_mapping: null,
     samples: [
         [
             "Юлия, съешь ещё этих мягких французских булок из Йошкар-Олы, да выпей алтайского чаю",
-            "Yuliya, s``esh` eshhyo е`tix myagkix franczuzskix bulok iz Joshkar-Oly`, da vy`pej altajskogo chayu",
+            "Yuliya, s``esh` eshhyo e`tix myagkix franczuzskix bulok iz Joshkar-Oly`, da vy`pej altajskogo chayu",
```
Comment (corporateanon) [inline src/generated/gost_779_alt.ts:46]: Strange

File: src/generated/_definitions.ts [hunk 5]
```diff
@@ -33,13 +33,13 @@ export default [
...
-    gost_779,
-    gost_779_alt,
-    gost_7034,
     gost_16876,
     gost_16876_alt,
     gost_52290,
     gost_52535,
+    gost_7034,
+    gost_779,
+    gost_779_alt,
```
Comment (corporateanon) [inline src/generated/_definitions.ts:42]: The order does not matter here, does it?

File: src/generated/gost_779_alt.ts (hunk 1 above)
Comment (nalgeon) [reply src/generated/gost_779_alt.ts:36]: It's OK

File: src/generated/yandex_maps.ts (hunk 2 above)
Comment (nalgeon) [reply src/generated/yandex_maps.ts:40]: It' OK, the schema was updated recently

File: src/generated/yandex_maps.ts (hunk 3 above)
Comment (nalgeon) [reply src/generated/yandex_maps.ts:46]: It' OK, the schema was updated recently

File: src/generated/gost_779_alt.ts (hunk 4 above)
Comment (nalgeon) [reply src/generated/gost_779_alt.ts:46]: It's OK

File: src/generated/_definitions.ts (hunk 5 above)
Comment (nalgeon) [reply src/generated/_definitions.ts:42]: Absolutely

**Comment by nalgeon** [comment]
The JSON schemas are the source of truth, you can trust them.

File: src/generated/gost_779_alt.ts (hunk 1 above)
Comment (makc) [reply src/generated/gost_779_alt.ts:36]: nice

**Pull request merged**